cli-commands/nodeos/subcommands/stop/messages.js

```diff
@@ -0,0 +1,21 @@
+const chalk = require('chalk');
+
+module.exports = {
+    'StoppingNodeos': () => { console.log(chalk.magentaBright('===== Stopping nodeos ... =====')); },
+    'NoRunningNodeos': () => { console.log(chalk.redBright('===== There is no running nodeos ... =====')); },
+    'SuccessfulProcessTermination': () => { console.log(chalk.blueBright('===== Instance terminated =====')); },
```

Comment:
SuccessfulProcessTermination -> SuccessfulStop

Reply:
removed, instead will be used message SuccessfullyStopped